index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link href="./style.css" rel="stylesheet">
+  </head>
+  <main>
+  <body>
+    <div>
+      <h1 id="title">Project Management Survey Form</h1>
+      <p id="description">
+        <em>Thank you for taking the time to improve the platform</em>
+      </p>
+    </div>
+      <form id="survey form">
+          <div>
+        <label for="name">Name</label><br />
+        <input
+          type="text"
+          id="name-label"
+          placeholder="Enter your name"
+        /><br /><br />
+        <label for="email">Email:</label><br />
+        <input
+          type="text"
+          id="email-label"
+          placeholder="Enter your Email"
+        /><br /><br />
+        <label for="number">Age(optional)</label><br />
+        <input type="number" id="number-label" min="10" max="70" />
+        <br /><br />
+        <label for="dropdown"
+          >Which option best describes your current role?</label
+        ><br />
+        <select
+          current
+          role="dropdown"
+          id="dropdown"
+          placeholder="select your current role"
+          style="width: 150px"
```

Comment:
I think its best practice to write clean code for clarity and readability

Reply:
- [ ] We can improve this review message by being more descriptive and giving a reason for the specifically requested change not general. ex: by saying please remove `<br>` tag and use padding and margin to make your code readable.

Reply:
Looks like you forgot to be more specific in the tag you're asking to remove, :mag: remember that it could be obvious from an external point of view, but sometimes we just don't see the issue, even when it's in front of our eyes :open_mouth: 

> remove the tag and...

Reply:
i did put the tag, but don't know why its not visible as soon as i click on the start review button